index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <script src="https://kit.fontawesome.com/8c36836212.js" crossorigin="anonymous"></script>
+        <title>Youtube</title>
+    </head>
+
+    <body>
+        <header>
+            <nav>
+                <ul class="menu">
+                    <li><i class="fa-solid fa-house"></i>Home</li>
+                    <li><i class="fa-solid fa-id-card-clip"></i>Shorts</li>
+                    <li><i class="fa-solid fa-hand-pointer"></i>Subscriptions</li>
+                    <li><i class="fa-solid fa-bookmark"></i>Library</li>
+                    <li><i class="fa-solid fa-clock-rotate-left"></i>History</li>
+                </ul>
```

Comment:
I think your menu will look better if you added the class "menu" and gave it "list-style: none"

.menu { 
    list-style: none;
    }